src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -185,6 +189,35 @@ public function isInitializable(string $class, string $property, array $context
         return false;
     }
 
+    private function extractFromDeclaredType(string $class, string $property)
+    {
+        // to be removed as soon as Symfony bumps the minimum PHP Version to 7.4
+        if (version_compare(PHP_VERSION, '7.4.0-dev', '<')) {
+            return null;
+        }
+
+        try {
+            $reflectionClass = new \ReflectionClass($class);
+        } catch (\ReflectionException $e) {
+            return null;
+        }
+
```

Comment:
To prevent test fails you can add following code here:
```php
if (!$reflectionClass->hasProperty($property)) {
    return null;
}
```

Reply:
Or you can expand try-catch block for the whole function content (to prevent errors, which can happen while calling `hasType`, `getProperty`, `getType` etc. which can throw ReflectionException)

Reply:
I've used the same `try/catch` block where I create the reflection class to get the property, and if it doesn't exist, it'll return `null`